meta.go

```diff
@@ -534,6 +542,13 @@ func main() {
 			Name:  "get",
 			Usage: "Get a metadata with key",
 			Action: func(c *cli.Context) error {
+				// Ensure that the CLI is concurrency safe. Get may write if fetching lastSuccessful; lock exclusively.
+				flocker := flock.New(lockfile)
+				if err := flocker.Lock(); err != nil {
+					failureExit(err)
```

Comment:
are clients expected to re-try if it can't acquire lock ? Should there be any sort of auto retry after a brief sleep ?

Reply:
well… this isn't a call to TryLock, so if it fails, its something more dastardly… that said… perhaps we can use something that we'd more likely need or have access to be locking… I just tried and didn't have access to create the lock file in `/var/run` on one machine… So… perhaps we can put it in the "meta-space".

Reply:
(put another way, this will wait until it _CAN_ acquire a lock; and only fail due to file access issues).